modules/creating-a-project-workbench.adoc

```diff
@@ -23,14 +23,18 @@ The *Data science projects* page opens.
 . Click the name of the project that you want to add the workbench to.
 +
 The *Details* page for the project opens.
-. Click *Create workbench* in the *Workbenches* section.
+. In the *Workbenches* section, click *Create workbench*.
 +
 The *Create workbench* page opens.
 . Configure the properties of the workbench you are creating.
-.. Enter a *name* for your workbench.
-.. Enter a *description* for your workbench.
-.. Select the *notebook image* to use for your workbench server.
-.. Select the *container size* for your server.
+.. In the *Name* field, enter a name for your workbench.
+.. Optional: In the *Description* field, enter a description to define your workbench.
+.. In the *Notebook image* section, complete the relevant fields to specify the most suitable notebook image to use with your workbench. 
```

Comment:
Is there a reason we need to include 'the most suitable' or 'relevant fields' here? The data scientist will likely have an idea about what technologies they require for their data science work. Because of this, they will need to select a notebook image that is most suitable for the type of work that they plan to carry out. Some notebook images will not be suitable for their work.

Are we keeping things broad with 'complete the relevant fields' because there are different fields for the different notebook images? The fields are the same. In this case, the "Notebook Image" section is just the title of a section.

Reply:
Ok, understood. In that case, I do think we could consider cutting this down as being a bit fluffy. 

From what I've seen of the docs, we assume the user is going to select what they think is most suitable/relevant for every step. It could cause confusion to only call out relevance/suitability on certain steps (or imply that selecting the most suitable/relevant choice _doesn't_ matter on others).

Reply:
Fair point. I'll try and cut out the fluff.